README.md

```diff
@@ -263,20 +274,25 @@ Returns the currently logged-in user's Facebook user ID.
 ---------------------------------------
 
 <a name="sendMessage" />
-### api.sendMessage(message, thread_id, [callback])
+### api.sendMessage(message, threadID, [callback])
 
 Sends the given message to the thread_id.
 
 __Arguments__
 
-* `message` - A string (for backward compatibility) or a message object as described below.
-* `thread_id` - A string, number, or array representing a thread. It happens to be someone's userId in the case of a one to one conversation or an array of userIds when starting a new group chat.
-* `callback(err, threadID)` - A callback called when sending the message is done (either with an error or with an confirmation object). `threadID` contains only the thread_id where the message was sent.
+* `message`: A string (for backward compatibility) or a message object as described below.
```

Comment:
I think we can still overload the function and remove the (for backward compatibility)